src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -485,7 +492,7 @@ object ScalafixPlugin extends AutoPlugin {
                 case jar =>
                   Seq(jar)
               }
-            write(files.map(FileInfo.lastModified.apply))
+            write(files.map(Hash(_)))
```

Comment:
Could you document the reason for using this rather than hashStyle here (as a separate commit)? Thanks!

Reply:
I am still iterating but i will as soon as my build is green